### PR TITLE
Follow up PR

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jazz-build-module/environment-deployment-metadata-loader.groovy [hunk 1]
```diff
@@ -76,10 +76,9 @@ def getRepoURL() {
 }
 
 def getEnvironmentLogicalId() {
-	if (g_environment_logical_id == null) {
-		def token = g_login_token
+	if (g_environment_logical_id == null && service_config['domain'] != "jazz") {
```
Comment (suryajak) [inline jazz-build-module/environment-deployment-metadata-loader.groovy]: Why are we having to exclude jazz namespace?

File: jazz-build-module/environment-deployment-metadata-loader.groovy [hunk 2]
```diff
@@ -150,10 +149,9 @@ def createPromotedEnvironment(environment_logical_id, created_by) {
 def checkIfEnvironmentAvailable(environment_logical_id) {	
 	def isAvailable = false
 	try {
-		if (environment_logical_id) {
-			def token = g_login_token
+		if (environment_logical_id && service_config['domain'] != "jazz") {
```
Comment (suryajak) [inline jazz-build-module/environment-deployment-metadata-loader.groovy]: Does this mean we can't have non-prod environments for jazz namespace?

File: jazz-build-module/environment-deployment-metadata-loader.groovy (hunk 1 above)
Comment (SiniWilson) [reply jazz-build-module/environment-deployment-metadata-loader.groovy]: While deploying jazz services , the environment handlers and apis also login may not deployed.

Comment (suryajak) [reply jazz-build-module/environment-deployment-metadata-loader.groovy]: ahh, good point. We need to find another way to work around that. Lets come up with alternative ways that are better than the current "if" clauses.

**Pull request merged**